test/styles/bibtex.js

```diff
@@ -0,0 +1,72 @@
+const assert = require('assert');
+const bibtexmisc = require('../../model/styles/bibtexMisc');
+const biblatexSoftware = require('../../model/styles/biblatexSoftware');
+
+const oneAuthorSourceData = require('../mockSourceData/allFieldsOneAuthor');
+const threeAuthorsSourceData = require('../mockSourceData/allFieldsThreeAuthors');
+
+describe('BibTexMiscStyle', () => {
+  describe('format', () => {
+    it('Month field, one author', () => {
+      let formattedString = bibtexmisc.format(oneAuthorSourceData);
```

Comment:
It should be run automatically but first you have to copy the hook from the `dev` directory in the project to your `.git/hooks` directory locally.  Details on how to do this is in the readme under Contributor Install.  If you have any questions or its not working let me know.